### ISSUE TITLE
Empty preview, various errors in devtools

### ISSUE THREAD

**Issue by CherryDT** [RandomFractals/vscode-data-preview#291]
I've just installed this extension and it looked great in the store, but it doesn't seem to work.

I've tried opening a CSV file and a JSON file, and both times I get an empty preview:

![image](https://user-images.githubusercontent.com/6010463/133388534-78aba67e-3af7-42ec-a2a0-0efc1487ddf8.png)

However, in the status bar I can see a number of rows:

![image](https://user-images.githubusercontent.com/6010463/133388708-ea5b6362-ad01-4471-bdc6-a5c1e4b5192c.png)

In the devtools there are various errors:

![image](https://user-images.githubusercontent.com/6010463/133388802-0ee74dd1-14eb-4193-8647-8b5bb7e07074.png)

![image](https://user-images.githubusercontent.com/6010463/133389091-753622bd-b163-40f8-be5d-5da415da884a.png)

![image](https://user-images.githubusercontent.com/6010463/133389167-b5920dfa-672d-4969-bf70-1f1d5d094289.png)

![image](https://user-images.githubusercontent.com/6010463/133389206-ed207ef7-1b73-40de-9375-f6c34215407d.png)

![image](https://user-images.githubusercontent.com/6010463/133389260-a97275a8-e7aa-453b-8901-46421fe68608.png)

![image](https://user-images.githubusercontent.com/6010463/133389279-2d5ab28d-398f-48ef-83d2-6a66ba3feff9.png)

![image](https://user-images.githubusercontent.com/6010463/133389304-332ee6c4-d5dd-422b-91a1-3f711470748b.png)

VSCode version:

```
Version: 1.59.1 (user setup)
Commit: 3866c3553be8b268c8a7f8c0482c0c0177aa8bfa
Date: 2021-08-19T11:56:46.957Z (3 wks ago)
Electron: 13.1.7
Chrome: 91.0.4472.124
Node.js: 14.16.0
V8: 9.1.269.36-electron.0
OS: Windows_NT x64 10.0.19043
```

**EDIT:** While I was playing around, I got it to work _once_. I tried maybe 50 times or so, but it worked only once!

**EDIT2:** I just discovered #250, somehow missed that at first. Yes, light theme fixes it for me too. Should my issue be closed? I'm not sure if it's all the same because the errors appear to be different...

**Comment by RandomFractals** [comment]
duplicate of #250
